DependencyInjection/Compiler/FormFactoryCompilerPass.php

```diff
@@ -11,10 +11,10 @@
 
 namespace Sonata\CoreBundle\DependencyInjection\Compiler;
 
-use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Bundle\FrameworkBundle\DependencyInjection\Compiler\FormPass;
```

Comment:
I'm not saying this PR adds a requirement that was not here before, but since we are making things explicit, we should probably add the requirement for the sake of completeless, even more because IMO, if you reference a FQCN, you should require the package that provides it (apart maybe from testing packages). Right now it works because :

```
composer why symfony/framework-bundle
friendsofsymfony/rest-bundle   2.1.1    requires  symfony/framework-bundle (^2.7|^3.0)  
jms/serializer-bundle          1.2.1    requires  symfony/framework-bundle (~2.3|~3.0)  
nelmio/api-doc-bundle          2.13.1   requires  symfony/framework-bundle (~2.3|~3.0)  
sensio/framework-extra-bundle  v3.0.25  requires  symfony/framework-bundle (~2.3|~3.0)
```